### ISSUE TITLE
[Feature Request] Add Provider data to Overseerr

### ISSUE THREAD

**Issue by Tharic99** [sct/overseerr#654]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I'd like to see if Overseerr can utilize the existing Provider data that TMD exposes here - https://developers.themoviedb.org/3/movies/get-movie-watch-providers

That data could then be used to add a badge on top of the display of the movie/show in the GUI below where the Movie/Series badge currently displays. So showing "Netflix" or "Disney+" or "Amazon Prime" or any other providers there. Then, in addition when going into the actual item itself, the data could also be displayed on the right hand side by studio.

**Describe alternatives you've considered**
None

**Comment by sct** [comment]
Not stale

**Comment by EmptyVisual** [comment]
@sct adding to this, I think this would be a fantastic addition to Overseerr, Reasons being I've listed below


- (the main one) Storage space is limited on certain Media setups - this will save a huge amount of space on the Administrators setup if the end user is being hinted that the media can be consumed elsewhere.
- This sets Overseerr apart from the rest (not just another request system) - Whilst Overseerr is already fantastic and just makes sense for a lot of things, other request management software doesn't have this functionality.
- A central platform for media discovery - not having to open TMDB or JustWatch on a second tab to check content availability 😜 
- Users can toggle which Services they would like to show (ie. what ones they're paying for so they know it's instantly available elsewhere if they want to watch it straight away) - Alternatively, Overseerr can show availability on streaming services based on the end-users Country (whether we get that from an API on the plex user or a configurable user option in Overseerr) 
- More fine-grained permissions control from an Administration level over what content can be requested (can be added later, after the basic functionality has been added first)
  - Administrators can restrict access to requests if it's available elsewhere
  - Users can opt-out if they wish to not request something if they know it's available elsewhere


Just taking a look at the endpoints available on the TMDB API docs, we should have what we need. Hopefully, the points I've mentioned are worth considering such functionality. Thanks for your time and effort in this project!
